srcpkgs/ppp/template

```diff
@@ -26,51 +26,23 @@ conf_files="
  /etc/ppp/chap-secrets
  /etc/ppp/pap-secrets"
 
-CFLAGS="-D_GNU_SOURCE"
-
-if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
-	makedepends+=" musl-legacy-compat"
-fi
-
 do_configure() {
-	# Custom CFLAGS.
-	find -name "Makefile.linux" -exec sed -i "{}" \
-		-e "s;CFLAGS\\s*=\(.*\);CFLAGS = ${CFLAGS} \1;" \;
-	# Custom LDFLAGS for chat, pppoe-discovery and pppdump
-	sed -i chat/Makefile.linux \
-		-e "s;-o chat ;${LDFLAGS} -o chat ;"
-	sed -i pppd/plugins/rp-pppoe/Makefile.linux \
-		-e "s;-o pppoe-discovery ;${LDFLAGS} -o pppoe-discovery ;"
-	sed -i pppdump/Makefile.linux \
-		-e "s;-o pppdump ;${LDFLAGS} -o pppdump ;"
-	# Enable active filter
-	sed -i "s:^#FILTER=y:FILTER=y:" pppd/Makefile.linux
-	# Enable ipv6 support
-	sed -i "s:^#HAVE_INET6=y:HAVE_INET6=y:" pppd/Makefile.linux
-	# Enable Microsoft Proprietary Callback Control Protocol
-	sed -i "s:^#CBCP=y:CBCP=y:" pppd/Makefile.linux
```

Comment:
This one is still disabled by default. You can try building with `CBCP=y` in `do_build` instead of doing this sed dance, though.

Reply:
Thanks for pointing that out. Updated.